app/Http/Transformers/PostTransformer.php

```diff
@@ -69,7 +69,10 @@ public function transform(Post $post)
      */
     public function includeSignup(Post $post)
     {
-        return $this->item($post->signup, new SignupTransformer);
+        // Don't include posts but include the user.
```

Comment:
I'm not sure I understand this comment. You might not need it anyway because I think the code is pretty clear!